### DISCUSSION TITLE
Imported assets show as recently loaded files

### DISCUSSION THREAD

**Discussion by PascalLadalle** [4ian/GDevelop#3659] [Enhancement ideas (and their technical discussions)]
When right-clicking on GDevelop's icon in the start bar, it shows recently loaded jsons which is very convenient for opening another instance of GD with an example project, for instance. But it also shows recently imported assets, which is quite useless IMO.
Would it be possible to filter those out?
![image](https://user-images.githubusercontent.com/25978830/105083558-5c4b5580-5a95-11eb-88be-45f78ddbe809.png)

Windows 10 64-bit

**Comment by 4ian** [comment]
Ah interesting I did not even know this was done! I guess it's an automatic feature of Electron, that we should change to plug it to the methods doing the file openings :)

**Comment by Bouh** [comment]
By default all resources added in the app will be in this list.
I see nothing on Electron docs to filter by an extension file.

Electron give a way to [clear and set the paths.](https://www.electronjs.org/docs/api/app#appaddrecentdocumentpath-macos-windows)
We can easily use the fonctions added in this [PR "Open recent project"](https://github.com/4ian/GDevelop/pull/1762)  for get the right paths, clear the list, and add the correct paths.

Shouldn't be hard to add IMO

**Comment by INNOVATIVEGAMER** [comment]
https://github.com/4ian/GDevelop/blob/ff55b8b6a05c3ce7821dc5e28b77f3779b7f3a4a/newIDE/app/src/MainFrame/ElectronMainMenu.js#L362-L367

> Shouldn't be hard to add IMO

@Bouh Do you mean by changing the code here as follows will fix the issue ?

if (electron) electron.clearRecentDocuments();
      const recentFiles: Array<FileMetadataAndStorageProviderName> = preferences.getRecentProjectFiles();
      if (recentFiles.length > 0)
        buildAndSendMenuTemplate(project, i18n, recentFiles, shortcutMap);

**Comment by 4ian** [comment]
I've not thought about it completely, maybe that could work here, not sure if we should reuse this component or make another.

**Comment by AdeshKhandait** [comment]
Hi!!! I actually want to solve the issue but i don't from where to start and how to start i am new to community

**Comment by 4ian** [comment]
Hi, please start by following the README that explain everything to get started. Then investigate a bit the codebase. I unfortunately don't have the time to explain to everyone what to do precisely, but that's actually part of the challenge! Look in the codebase, find stuff, experiment, until you have an idea ;)

**Comment by AdeshKhandait** [comment]
Thanks for your advice i will do as you say!!!!

**Comment by 85ayush** [comment]
I would love to work on this issue, I am new to open source contribution, can you please guide me?
